src/Components/Restart/GameRestart.js

```diff
@@ -0,0 +1,19 @@
+import React, { useContext } from 'react';
+import { UserContext } from '../../Context/GameContext.js';
+import './GameRestart.css';
+
+export default function GameRestart({ space }) {
+  const { gameRestart } = useContext(UserContext);
+  return (
+    <>
+      <button
+        className="button"
+        onClick={() => {
+          gameRestart(space);
```

Comment:
when you call this function here, you're passing a parameter ```space``` but when you define the function you don't define it with any arguments so ```space``` here is ignored since you haven't told the function how to handle arguments